RELEASE_NOTES.rst

```diff
@@ -9,8 +9,10 @@ ARTIQ-7
 Highlights:
 * WRPLL
 * ``get()``, ``get_mu()``, ``get_att()``, and ``get_att_mu()`` functions added for AD9910 and AD9912
+* Phaser coredevice driver: refactor, bug fixes and added documentation
```

Comment:
This is not a refactor. The important point that users need to be aware of for their existing code is that the defaults and the behavior changes. Added documentation and vague "bug fixes" doesn't mean anything for existing code.

Reply:
Sure